src/test/ui/rfc-2457/idents-normalized.rs

```diff
@@ -0,0 +1,8 @@
+// check-pass
+#![feature(non_ascii_idents)]
+
+struct Résumé; // ['LATIN SMALL LETTER E WITH ACUTE']
+
+fn main() {
+    let _ = Résumé; // ['LATIN SMALL LETTER E', 'COMBINING ACUTE ACCENT']
```

Comment:
Some folks in the hypothetical future might run rustfmt on all tests. And there were talks about rustfmt doing normalization. I wonder whether there should be some verification that these literals indeed contain those characters otherwise the test might be rendered ineffective in the future.